lib/brain.js

```diff
@@ -1949,6 +1973,11 @@ Brain.prototype.smsFlowHandleReturnState = function smsFlowHandleReturnState ()
     return this.runComplianceTiers(nonCompliantTiers)
   }
 
+  // verify if any non compliant manual trigger was scanned and end tx
+  if (this.txBlockedByManualTrigger) {
+    this.txBlockedByManualTrigger = false
```

Comment:
Hmm, if I recall correctly, showing the blocked customer screen always leads to calls to `_idle()`. Is it possible to move this `txBlockedByManualTrigger` reset and all its instances (lines 1978, 2204, 2747) to the `_idle()` function instead? It'll just be called once and it's closer to the other variable resets